### DISCUSSION TITLE
NASA Earthdata Cloud Cookbook

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [hytest-org/hytest#259] [General]
Just learned about this at the ESIP winter meeting.   Just FYI:
https://nasa-openscapes.github.io/earthdata-cloud-cookbook/

**Comment by amsnyder** [comment]
We should collect helpful references somewhere other than issues (issues should be a problem that will be solved and closed). @gzt5142 - do you think we should lump this in with your ['Foundations' material](https://github.com/hytest-org/hytest/blob/main/essential_reading/ReadingsTutorials.md)? Or should we focus that on the packages we use and create a new markdown file for useful references?
